src/utils/utils.ts

```diff
@@ -36,19 +37,19 @@ export function errorDataMessage(error: Exception | Error | any): string {
     return `${errorMessage}`;
 }
 
-export function tryPromise<T>(p: Promise<any>, exceptionType: ExceptionType, message: string) {
+export function tryPromise<T>(p: Promise<any>, exceptionType: ExceptionType, message: string, call: ExtendedAppCallRequest) {
     return p.
         then((response) => {
             return <T>response.data;
         }).
         catch((error) => {
             const errorMessage: string = errorDataMessage(error);
-            throw new Exception(exceptionType, `${message} ${errorMessage}`);
+            throw new Exception(exceptionType, `${message} ${errorMessage}`, call, error);
```

Comment:
Since we simply "favor" `error` over `message` in the code line below, message will never be used:

https://github.com/liz-segura98/mattermost-app-google-drive/blob/5a10b34287451dc45407ed4239f0fa02eaa8057f/src/utils/exception.ts#L17

Since the message is already factoring in the error, do we need the `error` parameter at all? We would lose the context of `message` with the current code as well

Reply:
This issue is resolved in commit [350239a](https://github.com/liz-segura98/mattermost-app-google-drive/pull/2/commits/350239a58a8e8aea5349f8844e1275750e92cec6)!